bundler/cli.js

```diff
@@ -24,7 +24,13 @@ function add(value) {
 	command = null
 }
 
-bundle(params.input, params.output, {watch: params.watch})
+try {
+	bundle(params.input, params.output, {watch: params.watch})
```

Comment:
Better idea: let's just not catch it (i.e. rip out the `try`/`catch` altogether). If we can't create the bundle in the first place, why should we try to minify it?